src/js/stores/export.js

```diff
@@ -128,15 +128,22 @@ define(function (require, exports, module) {
          */
         _assetUpdated: function (payload) {
             var documentID = payload.documentID,
-                layerIDs = Array.isArray(payload.layerIDs) ? payload.layerIDs : [payload.layerIDs];
+                layerIDs = payload.layerIDs;
 
             if (!documentID) {
                 throw new Error ("Can not update an asset without a valid documentID (%s)", documentID);
             }
 
             var curDocumentExports = this.getDocumentExports(documentID) || new DocumentExports(),
-                nextDocumentExports = curDocumentExports.mergeLayerAssets(layerIDs, payload.assetPropsArray);
+                nextDocumentExports;
 
+            if (layerIDs || layerIDs === 0) {
```

Comment:
sorry, but would like to defer this.  In the next PR, this handler accepts an iterable of layerIDs.